jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/ResultFixtures.java

```diff
@@ -55,14 +55,28 @@ public static Result booleanFalseResult() {
 	}
 
 	public static Result numericResult() {
+		return numericResult(10);
+	}
+	public static Result numericResult(Object numericValue) {
 		return new Result(
 				0,
 				"ObjectPendingFinalizationCount",
 				"sun.management.MemoryImpl",
 				"ObjectDomainName",
-				"ObjectPendingFinalizationCount",
+				"MemoryAlias",
```

Comment:
The isssue I see there is that the previous naming mostly matched what can actually be found in a JVM, while the new one does not make sense. I see the point in having an alias that is different to provide better test, but we should probably keep something that make sense in this context. For example: resultAlias : object_pending_finalization_count.